tests/trainer/test_trainer.py

```diff
@@ -661,3 +661,196 @@ def on_batch_start(self, trainer, pl_module):
     assert not trainer.interrupted
     trainer.fit(model)
     assert trainer.interrupted
+
+
+def test_trainer_config_cpu():
```

Comment:
can we parametrize these test per no-GPU, 1-GPU, multi-GPU?
happy to write a suggestion about it... (ping me back)

Reply:
Done